graylog2-web-interface/src/pages/ShowDashboardPage.jsx

```diff
@@ -119,6 +119,9 @@ const ShowDashboardPage = React.createClass({
   _dashboardIsEmpty(dashboard) {
     return dashboard.widgets.length === 0;
   },
+  _validDimension(dimension) {
+    return dimension !== null && dimension !== undefined && dimension !== 0;
```

Comment:
Shouldn't this be as follows (using [`Number.isInteger()`](https://www.w3schools.com/jsref/jsref_isinteger.asp)):
```javascript
return Number.isInteger(dimension) && dimension > 0;
```